src/Client/css/im.css

```diff
@@ -86,7 +86,7 @@ body {
 
 .profile-description p:last-of-type,
 .description-message p:last-of-type {
-    margin-bottom: 0px;
+    margin-bottom: 25px;
```

Comment:
Forgot to mention this in my commit message, but this deals with the `about` section having bottom ~padding~ margin